libr/util/print.c

```diff
@@ -1,5 +1,6 @@
 /* radare - LGPL - Copyright 2007-2020 - pancake */
 
+// WTF :D
```

Comment:
remove

Reply:
i prefer to fix the incllude instead of removing the comment